### ISSUE TITLE
Use argparse instead of manually (un)commenting scripts

### ISSUE THREAD

**Issue by JustinStarreveld** [JustinStarreveld/ROBIST#2]
https://github.com/JustinStarreveld/ROBIST/blob/640933981d8bd18453a7914dd3074b4429c9e302/examples/wdp_experiments.py#L236-L243

**Comment by sjvrijn** [comment]
For example, see [here](https://github.com/sjvrijn/multi-level-co-surrogates/blob/b2110b6a38ec965489afd63b1d4949ec4be2c094/scripts/processing/2022-06-15-polar-plot-gradients.py#L155-L171):

```python
parser = argparse.ArgumentParser()
parser.add_argument("--exts", action="extend", nargs="+", type=str,
                    help="File extensions to use when saving images. Default: [.PNG, .PDF].")
parser.add_argument('--force-regen', action='store_true',
                    help="Force regeneration of all caching files. Default: --no-force-regen")
```

**Comment by sjvrijn** [comment]
closed by 831ee95
